sup3r/pipeline/forward_pass.py

```diff
@@ -46,9 +44,9 @@ class ForwardPassStrategy(InputMixIn):
 
     def __init__(self, file_paths, model_args, s_enhance, t_enhance,
                  fwp_chunk_shape, spatial_pad, temporal_pad,
+                 temporal_slice=slice(None),
```

Comment:
while we're editing this could we add the hr_spatial_coarsen arg to pass to the data handler?

Reply:
Yeah can do, whats the use case? Would this be for doing validation with data thats already high res?

Reply:
Yeah that exactly.

Reply:
cool, done!